etebase_server/settings.py

```diff
@@ -163,7 +163,17 @@
     if "database" in config:
         DATABASES = {"default": {x.upper(): y for x, y in config.items("database")}}
 
-ETEBASE_CREATE_USER_FUNC = "django_etebase.utils.create_user_blocked"
```

Comment:
This line shouldn't be removed, shouldn't it? Or is it a merge issue?

Reply:
It should be fixed now.